README.md

```diff
@@ -23,6 +23,7 @@ This extension controller supports the following Kubernetes versions:
 
 | Version         | Support     | Conformance test results |
 | --------------- | ----------- | ------------------------ |
+| Kubernetes 1.26 | 1.26.0+     | N/A |
```

Comment:
```suggestion
| Kubernetes 1.27 | 1.27.0+     | N/A |
```

Reply:
Done in https://github.com/gardener/gardener-extension-provider-alicloud/pull/609